src/scss/_clarity-overrides.scss

```diff
@@ -1,6 +1,6 @@
 // === *** OVERRIDES *** ===
 .header {
-  z-index: 1;
+  z-index: 4;
```

Comment:
Why 4? 🤔

Reply:
![z-index-stack-level](https://user-images.githubusercontent.com/10655408/54280366-9c874600-4597-11e9-8130-f5abfa8041b7.png)

Each CodeMirror Line (`.CodeMirror pre`) has a z-index of 2. and the **Download** button (`.query-result__bottom-actions`) has an index of 3

Reply:
👍